proxy.conf.json

```
@@ -1,6 +1,6 @@
 {
     "/cgi-bin/*": {
-        "target": "https://sng107.sng.uni-frankfurt.de/dev/",
+        "target": "http://dbsvm.mathematik.uni-marburg.de:32800/",
```

Please don't commit these config file defaults
```suggestion
        "target": "https://sng107.sng.uni-frankfurt.de/dev/",
```